homeassistant/components/webostv/translations/ca.json

```diff
@@ -31,8 +31,7 @@
     },
     "options": {
         "error": {
-            "cannot_retrieve": "No es pot obtenir la llista de fonts. Assegura't que el dispositiu est\u00e0 enc\u00e8s",
-            "script_not_found": "No s'ha trobat l'script"
+            "cannot_retrieve": "No es pot obtenir la llista de fonts. Assegura't que el dispositiu est\u00e0 enc\u00e8s"
```

Comment:
The core repo is not the source of truth for the translations so this will not have any effect. Lokalise needs to be updated to remove old strings. I think there's a script that does this, but you need admin access to Lokalise I believe.

Reply:
I was not sure what is the right way to do it, should I adjust this PR to remove it only from `strings.json` and `en.json` and ask an admin to remove the translations Lokalise?

Reply:
Once merged, ping me, I'll run the cleanup.